<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCylinderDialog</class>
 <widget class="QDialog" name="NewCylinderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Cylinder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QSpinBox" name="segmentASpinBox">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="segmentHLabel">
     <property name="text">
      <string>Segment H:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="segmentALabel">
     <property name="text">
      <string>Segment A:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="radiusDoubleSpinBox">
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>X:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="axisLabel">
     <property name="text">
      <string>Axis:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="axisComboBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="segmentRSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="yLabel">
     <property name="text">
      <string>Y:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="zLabel">
     <property name="text">
      <string>Z:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="yDoubleSpinBox">
     <property name="minimum">
      <double>-9999.989999999999782</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>-9999.989999999999782</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
     <property name="minimum">
      <double>-9999.989999999999782</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="segmentRLabel">
     <property name="text">
      <string>Segment R:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="segmentHSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>Radius:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="heightDoubleSpinBox">
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>80.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="heightLabel">
     <property name="text">
      <string>Height:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewCylinderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewCylinderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
